src/main/java/com/commercetools/project/sync/SyncerFactory.java

```diff
@@ -90,7 +93,9 @@ private void closeClients() {
 
   @Nonnull
   CompletionStage<Void> sync(
-      @Nullable final String syncOptionValue, @Nullable final String runnerNameOptionValue) {
+      @Nullable final String syncOptionValue,
+      @Nullable final String runnerNameOptionValue,
+      final boolean fullSyncOptionValue) {
```

Comment:
why not isFullySync for naming the fullSyncOptionValue ?

Reply:
nice catch. No reason. I just missed it.

Reply:
fa6c4db